build_backup/backup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+echo "--------------- Check required parameters ---------------"
+_req_envs="
+Required parameters: \n
+BACKUPDIR: $BACKUPDIR \n
+ZOOKEEPERDATADIR: $ZOOKEEPERDATADIR \n
+BACKUPS_TO_KEEP: $BACKUPS_TO_KEEP \n
+"
+echo -e "$_req_envs"
+
+if [ -z "$BACKUPDIR" ] || [ -z "$ZOOKEEPERDATADIR" ] || [ -z "$BACKUPS_TO_KEEP" ]; then
+  echo -e "Some required env variables aren't defined.\n"
+  exit 1
+fi
+
+# Create backup directory if absent.
+# ----------------------------------
+echo "--------------- Check backup/tmp dirs ---------------"
+if [ ! -d "$BACKUPDIR" ] && [ ! -e "$BACKUPDIR" ]; then
+    mkdir -p "$BACKUPDIR"
+else
+    printf "Backup directory $BACKUPDIR is existed.\n"
+fi
+
```

Comment:
It would be nice to verify that `BACKUPS_TO_KEEP` contains a (reasonable) integer